app/scripts/views/settings/account_recovery/recovery_key.js

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const $ = require('jquery');
+const Cocktail = require('cocktail');
+const BaseView = require('../../base');
+const ModalSettingsPanelMixin = require('../../mixins/modal-settings-panel-mixin');
+const Template = require('templates/settings/account_recovery/recovery_key.mustache');
+const RecoveryKeyPrintTemplate = require('templates/settings/account_recovery/recovery_key_print.mustache');
+
+const {t} = BaseView;
+
+const View = BaseView.extend({
+  template: Template,
+  className: 'account-recovery-key',
+  viewName: 'settings.account-recovery.recovery-key',
+
+  events: {
```

Comment:
Like above, I can access this page w/o having set a recovery key just by browing to `/settings/account_recovery/recovery_key`. Maybe in a `beforeRender` or `afterRender`, ensure the user *has* a recovery key or else send them to the setup page.

Reply:
Added!